### PR TITLE
OSDOCS#9420: Removing references to deprecated deploymentconfigs from…

### PR DESCRIPTION
**Pull request by bergerhoffer**
… welcome page

Version(s):
4.14+

Issue:
https://issues.redhat.com/browse/OSDOCS-9420

Link to docs preview:
https://70831--ocpdocs-pr.netlify.app/openshift-enterprise/latest/welcome/#developer-activities

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@atiratree Can you please take a quick look?

Preview: https://70831--ocpdocs-pr.netlify.app/openshift-enterprise/latest/welcome/#developer-activities

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM

**Comment by bergerhoffer** [comment]
@kasturinarra Can you PTAL as well?

Preview: https://70831--ocpdocs-pr.netlify.app/openshift-enterprise/latest/welcome/#developer-activities

File: welcome/index.adoc [hunk 1]
```diff
@@ -197,8 +197,7 @@ xref:../applications/working_with_helm_charts/understanding-helm.adoc#understand
...
 - **xref:../openshift_images/index.adoc#overview-of-images[Create container images]**: A container image is the most basic building block in {product-title} and Kubernetes applications. By defining image streams, you can gather multiple versions of an image in one place as you continue to develop the image stream. With S2I containers, you can insert your source code into a base container. The base container is configured to run code of a particular type, such as Ruby, Node.js, or Python.
 
-- **xref:../applications/deployments/what-deployments-are.adoc#what-deployments-are[Create deployments]**:  Use `Deployment` and `DeploymentConfig` objects to exert fine-grained management over applications.
- xref:../applications/deployments/managing-deployment-processes.adoc#deployment-operations[Manage deployments] by using the *Workloads* page or OpenShift CLI (`oc`). Learn xref:../applications/deployments/deployment-strategies.adoc#deployment-strategies[rolling, recreate, and custom] deployment strategies.
+- **xref:../applications/deployments/what-deployments-are.adoc#what-deployments-are[Create deployments]**:  Use `Deployment` objects to exert fine-grained management over applications.
```
Comment (atiratree) [inline welcome/index.adoc:200]: looks good as is, but could be enhanced

```suggestion
- **xref:../applications/deployments/what-deployments-are.adoc#what-deployments-are[Create deployments]**:  Use `Deployment` objects to exert fine-grained management over applications. Deployments create replica sets according to the rollout strategy, which orchestrate pod lifecycles.
```

File: welcome/index.adoc [hunk 2]
```diff
@@ -229,7 +228,7 @@ ifdef::openshift-dedicated[]
 
 - *Create container images*: A container image is the most basic building block in {product-title} (and Kubernetes) applications. By defining image streams, you can gather multiple versions of an image in one place as you continue its development. With S2I containers, you can insert your source code into a base container that is set up to run code of a particular type (such as Ruby, Node.js, or Python).
 
-- *Create deployments*:  Use `Deployment` and `DeploymentConfig` objects to exert fine-grained management over applications. Use the Workloads page or `oc` CLI to manage deployments. Learn rolling, recreate, and custom deployment strategies.
+- *Create deployments*:  Use `Deployment` objects to exert fine-grained management over applications.
```
Comment (kasturinarra) [inline welcome/index.adoc:231]: @bergerhoffer i think we could still keep the line `Use the Workloads page or `oc` CLI to manage deployments. Learn rolling, recreate, and custom deployment strategies` right as it talks about deployments itself.

File: welcome/index.adoc (hunk 1 above)
Comment (bergerhoffer) [reply welcome/index.adoc]: Updated

**Comment by bergerhoffer** [comment]
@atiratree @kasturinarra Updated, please take another look when you can. Thanks!

**Comment by atiratree** [comment]
lgtm

**Pull request merged**